community/cypher/cypher-compiler-2.2/src/main/scala/org/neo4j/cypher/internal/compiler/v2_2/ast/rewriters/collapseInCollections.scala

```diff
@@ -20,36 +20,41 @@
 package org.neo4j.cypher.internal.compiler.v2_2.ast.rewriters
 
 import org.neo4j.cypher.internal.compiler.v2_2.ast._
-import org.neo4j.cypher.internal.compiler.v2_2.{bottomUp, Rewriter}
+import org.neo4j.cypher.internal.compiler.v2_2.{Rewriter, bottomUp}
 
```

Comment:
I think I was assigned the please-comment-new-classes card last retro, so here we go. Please add a brief comment explaining the intent and purpose of this class